### PR TITLE
Allow to set syntax for traefik host rules

### PR DESCRIPTION
**Pull request by Jorg88**

### CODE REVIEW

File: src/mpyl/project.py [hunk 1]
```diff
@@ -349,9 +349,9 @@ def from_config(values: Optional[dict]):
         if not values:
             return None
         return TraefikAdditionalRoute(
-            name=values.get("name", {}),
-            middlewares=values.get("middlewares", {}),
-            entrypoints=values.get("entrypoints", {}),
+            name=values.get("name", ""),
```
Comment (Jorg88) [inline src/mpyl/project.py:352]: This is not related to the task, just fixing some wrong types that I noticed. Boyscout rule I guess

**Review by ptab — APPROVED** [review]
Pre-approved before you add a test 😉
